blocks/init/src/blocks/components/image/components/image-editor.js

```diff
@@ -16,6 +23,25 @@ export const ImageEditor = (props) => {
   `;
 
   return (
-    <img className={imageClass} src={url} alt="" />
+    <Fragment>
+      {!url ?
+        <MediaPlaceholder
+          icon="format-image"
+          onSelect={onChangeMedia}
+          accept={accept}
+          allowedTypes={allowedTypes}
+        /> :
+        <Fragment>
+          <Button
+            isSecondary
+            onClick={() => onChangeMedia({})}
```

Comment:
Does this have to be a callback? In the MediaPlaceholder you're just calling `onChangeMedia`, but here it's in the arrow function.